open_event/views/admin/models_views/events.py

```diff
@@ -60,3 +60,10 @@ def completed_events_view(self):
     def current_view(self):
         events = DataGetter.get_current_events()
         return self.render('/gentelella/admin/event/current_events.html', events=events)
+
+    @expose('/<int:event_id>/update/', methods=('GET','POST'))
```

Comment:
@aditya1702 shouldn't we change only on POST? Because using GET I believe it would be easily modifiable by anybody and giving permissions would be difficult. @rafalkowalski What do you think?